src/core/VisuallyHidden/VisuallyHidden.md

```diff
@@ -0,0 +1,19 @@
+The **VisuallyHidden** component allows you to hide content from sight while keeping it available for screen readers.
+
+```js
+import { VisuallyHidden, SearchInput } from 'suomifi-ui-components';
+<>
+  <span>There is a hidden element under this text.</span>
+  <VisuallyHidden>
+    This element will be read by screen readers even though it's
+    visually hidden
+  </VisuallyHidden>
+
+  <SearchInput
```

Comment:
Could we replace this with TextInput? It will be more simple after the upcoming breaking for SearchInput.

Reply:
Search input was a common example I saw people use online when talking about where VisuallyHidden styles are used. Text input not so much. That's why I opted for this.

Reply:
Let's keep this as discussed elsewhere. Just have to remember to update when introducing changes to SearchInput.